src/invidious/config.cr

```diff
@@ -84,6 +84,7 @@ class Config
 
   # Used to tell Invidious it is behind a proxy, so links to resources should be https://
   property https_only : Bool?
+  property login_only : Bool?
```

Comment:
It would be good to add a descriptive comment on what this does, similar to the others.